core/block_svg.js

```diff
@@ -13,66 +13,81 @@
 goog.module('Blockly.BlockSvg');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.ASTNode');
-goog.require('Blockly.Block');
-goog.require('Blockly.blockAnimations');
-goog.require('Blockly.blockRendering.IPathObject');
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.common');
-goog.require('Blockly.connectionTypes');
-goog.require('Blockly.constants');
-goog.require('Blockly.ContextMenu');
-goog.require('Blockly.ContextMenuRegistry');
-goog.require('Blockly.Events');
+const ASTNode = goog.require('Blockly.ASTNode');
+const Block = goog.require('Blockly.Block');
+const Comment = goog.require('Blockly.Comment');
+/* eslint-disable-next-line no-unused-vars */
+const Connection = goog.requireType('Blockly.Connection');
+const ContextMenu = goog.require('Blockly.ContextMenu');
+const ContextMenuRegistry = goog.require('Blockly.ContextMenuRegistry');
+const Coordinate = goog.require('Blockly.utils.Coordinate');
+/* eslint-disable-next-line no-unused-vars */
+const Debug = goog.requireType('Blockly.blockRendering.Debug');
```

Comment:
Could you include part of the namespace name in this one for clarity? Like `BlockRenderingDebug`?

Reply:
Done